myhpom/models/document.py

```diff
@@ -1,7 +1,10 @@
-import os, tempfile
+import os, tempfile, base64, uuid
 
 from .user import User
 from django.db import models
+from django.core.urlresolvers import reverse
+from django.conf import settings
+from django.utils.timezone import now
 from django.core.files.base import ContentFile
```

Comment:
This `ContentFile` can be removed as its unused.

Reply:
I need to start flake8ing as part of my regular workflow. :-)